reflex/components/datadisplay/moment.py

```diff
@@ -0,0 +1,26 @@
+"""Moment component for humanized date rendering."""
+
+from reflex.components.component import Component
+from reflex.vars import Var
+
+
+class Moment(Component):
+    """The Moment component."""
+
+    tag: str = "Moment"
+    library: str = "react-moment"
```

Comment:
this library is awesome and can even make time-based callbacks from the frontend!

we should definitely wrap more of the API or at least make an issue to follow up and wrap the remaining API. Having a half-wrapped component in the base i think would be confusing for people.

Reply:
Yeah, I was experimenting with it and forgot I'd added it, should definitely be part of a separate PR for Moment.